### ISSUE TITLE
Potentially find a way to control question order 

### ISSUE THREAD

**Issue by miabonardi** [LemmaLegalConsulting/docassemble-GBLS#9]
1. Maybe add questions that appear in every letter (ie clients name, clients address, attorneys name, attorneys address, etc) to an order block to control the order of those questions and everything else will come in a random order
2. Use signature final method to make sure signature comes last (ask Quinten for formatting)

**Comment by h-mcguire** [comment]
Using separate .YML's for each "form" might be a good option? I'm unsure how these interactions work and if the "master interview" would be able to pull all the gathered info from the questions asked so far. 

But I think if we If we can break them into separate files, and keep all the info gathered available to auto-populate from, I think it would allow complete control of questions order and not risk asking for the same piece of info twice.

**Comment by miabonardi** [comment]
> Using separate .YML's for each "form" might be a good option? I'm unsure how these interactions work and if the "master interview" would be able to pull all the gathered info from the questions asked so far.
> 
> But I think if we If we can break them into separate files, and keep all the info gathered available to auto-populate from, I think it would allow complete control of questions order and not risk asking for the same piece of info twice.

Absolutely! We use the method of a `main.yml` for the main order that refers to sub interview orders in the individual yml files for each form for several other projects. We also add a `shared.yml` for any custom questions (not AL) shared by all forms. 

See:
- https://github.com/LemmaLegalConsulting/docassemble-MOHUDEvictionProject
- https://github.com/LemmaLegalConsulting/docassemble-DangerLaw
- https://github.com/LemmaLegalConsulting/docassemble-ElderLaw
